src/coreclr/jit/valuenum.cpp

```diff
@@ -4602,8 +4602,9 @@ bool ValueNumStore::IsVNVectorZero(ValueNum vn)
     }
 #if FEATURE_HW_INTRINSICS
     VNFuncApp funcApp;
-    if (GetVNFunc(vn, &funcApp) && funcApp.m_arity == 1)
+    if (GetVNFunc(vn, &funcApp))
```

Comment:
This arity check is still required (because as we've discussed on XARCH the arity will be zero).

Reply:
Yea, I noticed I screwed that one up.